app/models/course/material/folder.rb

```diff
@@ -10,6 +10,12 @@ class Course::Material::Folder < ActiveRecord::Base
   # TODO: Remove this after schema_validations #21 was closed
   validates :name, uniqueness: { case_sensitive: false, scope: :parent_id }, if: :parent
 
+  def files=(files)
```

Comment:
`files_attributes` would expect folder to have many files, but in the case we are creating materials from files, so it won't work.